test/test_examples.py

```diff
@@ -75,3 +98,17 @@ def test_async_power_metering_dag():
     meter.publish(payload=payload)
     time.sleep(.3)
 #     assert atn.total_power_w == 2100
+
+
+def test_collect_temp_data():
+    """Verify Scada receives publication from TankWaterTempSensor"""
+    load_house.load_all(house_json_file='../test/test_data/test_load_house.json')
+    scada = ScadaRecorder(node=typing.cast(ShNode, ShNode.by_alias["a.s"]))
+    thermo = TankWaterTempSensor(node=typing.cast(ShNode, ShNode.by_alias["a.tank.temp0"]))
+    time.sleep(1)
+    thermo.terminate_sensing()
+    thermo.sensing_thread.join()
+    scada.terminate_scheduling()
+    scada.schedule_thread.join()
+    assert scada.num_received > 0
+    assert scada.num_received_by_topic["a.tank.temp0/gt.telemetry.110"] == scada.num_received
```

Comment:
When we update the versioning of the schema, the topic will change and this line will have to change. But that seems like a good thing. Plus it is helpful to actually see what the topic looks like.

On another note, I was wondering why the tests terminated without calling `scada.gw_consume_client.loop_stop()` (etc for all 4 mqtt client loops for scada, and both mqtt client loops for thermo)